src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -702,8 +702,22 @@ public void QuickFetch()
 
         public void QuickFetch(int changesetId)
         {
-            var changeset = Tfs.GetChangeset(changesetId, this);
-            quickFetch(changeset);
+            try
+            {
+                var changeset = Tfs.GetChangeset(changesetId, this);
+                quickFetch(changeset);
+            }
+            catch (Exception ex)
+            {
+                // Ignore exceptions "TF14098: Access Denied: User ??? needs
+                // Read permission(s) for at least one item in changeset ???."
+                if (!ex.Message.Contains("TF14098"))
```

Comment:
It would be nicer to check for the exact exception here: `Microsoft.TeamFoundation.VersionControl.Client.ResourceAccessException`. However, then I would need to include the Microsoft namespace here and I have the impression that would interfere the separation of concerns in the `git-tfs` project, no?

Reply:
> However, then I would need to include the Microsoft namespace here and I have the impression that would interfere the separation of concerns in the git-tfs project, no?

Right, `GitTfs/Core` shouldn't be able to see anything in the `Microsoft.TeamFoundation` namespace, though it's mostly related to packaging/versioning rather than separation of concerns. If the class is consistent, it'd be ok to do something like `ex.GetType().FullName == "Microsoft.TeamFoundation.VersionControl.Client.ResourceAccessException"`, though my preference is to stick with the check for TF14098 in the exception message.